packages/datadog-plugin-moleculer/src/util.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+// TODO: add ctx.params when nested object properties are deprecated
+
+function moleculerTags (broker, ctx, config) {
+  const service = ctx.service || {}
+  const action = ctx.action || {}
+  const meta = config.meta && ctx.meta
+
+  return {
```

Comment:
We can probably skip the object creation here by passing the tag object into `molecularTags`. (This is a non-blocking comment.)

Reply:
Let's leave it like this for now since most of the logic will be redone anyway when the plugin is rewritten with the new plugin system.